module/purpose/Soccer/data/config/NUgus3/Soccer.yaml

```diff
@@ -0,0 +1,9 @@
+# Controls the minimum log level that NUClear log will display
```

Comment:
Will this folder name work as `NUgus{number}`? I have only ever seen it as lowercase `nugus{number}`

Reply:
it won't work, it has to match the hostname, which is `nugus{number}`.